parts/linux/cloud-init/artifacts/components.json

```diff
@@ -8,512 +10,18 @@
         "1.8.22"
       ]
     },
+
+    //new
     {
-      "downloadURL": "mcr.microsoft.com/oss/kubernetes/metrics-server:*",
+      "name": "nvidia-cuda",
```

Comment:
What is the difference between "name" and "fullPackageName"?

Reply:
full Image Name = registryURL + fullPackageName
name is mainly for applying logic in the scripts. For example, 
```
if name=="nvidia-cuda", do A; 
if name =="nvidia-grid", do b;
```
Another option is to put fullPackageNames, instead of name in the if conditions above.
But I guess there could be cases that the 2 fullPackageNames happen to be the same but the registryURLs are different.
Then we need a name for the feature owner to define the logic separately.

Reply:
Oh i see, i was just a bit confused about the discrepancy between names here

Reply:
You are good. That's a good call out.